generate_site/generate.py

```diff
@@ -18,9 +19,16 @@
 df_org.fillna("", inplace=True)
 # TODO: Download images? 'Headshot image' is the column name
 
+# Assuming https://github.com/pyladies/global-conference/pull/60 gets merged
+with open("speakers.csv", encoding="utf-8-sig") as f:
```

Comment:
A new file `speakers.csv` has been created as part of this PR. Editing it and regenerating the site will update the Speakers section of the homepage. Keep in mind only the first 4 speakers are rendered in the homepage, but all speaker data can be used to implemented future features like #57